### PR TITLE
Implement #57 to support parameterized test cases

### PR DESCRIPTION
**Pull request by johnklee**
As title.

### CODE REVIEW

File: tests/unit/test_flist.py [hunk 1]
```diff
@@ -2,11 +2,12 @@
 import sys
 import os
 import re
-import random 
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../../fpu')))  # noqa
-from fp import *  # noqa
-import flist
-from flist import *  # noqa
+import random
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../../')))  # noqa
```
Comment (wkCircle) [inline tests/unit/test_flist.py:6]: I have tested the code from my local, and humbly think that we don't need the `sys.path.insert` line.

File: tests/unit/test_flist.py [hunk 2]
```diff
@@ -156,11 +163,26 @@ def test_gapi_frange(self):
...
+          expected_result=True),
+      dict(
+          testcase_name='case 2',
+          test_func=lambda e: e == 5,
+          expected_result=False),
+      dict(
+          testcase_name='case 3',
+          test_func=lambda e: e > 1,
+          expected_result=True),
+  )
```
Comment (wkCircle) [inline tests/unit/test_flist.py:179]: Nit: indention 2?

**Review by wkCircle — CHANGES_REQUESTED** [review]
Looks very awesome! Only small things that might need modifications.

File: tests/unit/test_flist.py (hunk 2 above)
Comment (johnklee) [reply tests/unit/test_flist.py:171]: Yep. You are right. But within Google, indention in this case is 4 and my internal editor caused this issue. I have fixed it by using 2. Thanks for the reminding.

File: tests/unit/test_flist.py (hunk 1 above)
Comment (johnklee) [reply tests/unit/test_flist.py]: That's right. Fixed by merging to latest master branch.

**Comment by johnklee** [comment]
> Looks very awesome! Only small things that might need modifications.

I made some changes correspondingly. Please help me to reconfirm again. Thanks!

**Review by wkCircle — APPROVED** [review]
LGTM

**Pull request merged**